### ISSUE TITLE
Different validated hours for Catalan 

### ISSUE THREAD

**Issue by jmontane** [mozilla/voice-web#1403]
Hi,
According to https://voice.mozilla.org/en/new , Catalan has 33 validated hours.
![validated-1](https://user-images.githubusercontent.com/339645/44463404-e8809680-a617-11e8-9829-c76a97934784.png)
But, according to https://voice.mozilla.org/en/languages , Catalan has 45 validated hours.
![validated-2](https://user-images.githubusercontent.com/339645/44463493-309fb900-a618-11e8-92d2-5b14582d1868.png)

Why this divergence? How many hours are validated for Catalan language?

**Comment by jf99** [comment]
Also, the actual meaning of the numbers is unclear. I assume "hours recorded" counts all audio data that has ever been submitted. But what is "hours validated"? Does it only count recordings that have 2 "yes"-votes? Or are "no"-voted recordings also included?

I noticed there are languages where the red counter is twice as high as the green one. Could this be caused by many "no"-votes? If yes, then I think a third counter is needed for all (yes+no) votes.

**Comment by Gregoor** [comment]
You're intuition is correct. We also want to add some FAQ and hover information for this. Note that this new homepage isn't launched yet, but rather a ressourceful user found it (:

I agree with your sentiment that validated could also mean "ones that got no-voted" (but indeed it doesn't). Maybe you're right and no-voted clips should somehow affect this graph as well.

cc @mbransn

**Comment by mbransn** [comment]
Indeed, the intuition is correct and here are the definitions we will be providing via FAQ + `i` hover when this rolls to production:

**Hours Recorded**
_The count of voice recording hours we have collected so far._

**Hours Validated**
_The count of voice recording hours that have been validated by 2 out of 3 users with a vote of “Yes”. These mark progress toward the overall project 10k hours goal._

No-voted clips do not provide progress toward the 10k hours goal and are not viable for the dataset, therefore they are not 'counted'.

>I noticed there are languages where the red counter is twice as high as the green one. Could this be caused by many "no"-votes? If yes, then I think a third counter is needed for all (yes+no) votes.

This should be due to having more recorded clips in the queue (red) than have been validated (green).

**Comment by Gregoor** [comment]
The original issue is now fixed (but I don't want to close this issue, because the discussion seems valuable).
